From 62d588b0c609de17df5b4d1ebfbc67c456267efc Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Sat, 2 Oct 2021 00:13:14 +0200
Subject: [PATCH] =?UTF-8?q?docs:=20Fix=20build=20with=20EDS=20=E2=89=A5=20?=
 =?UTF-8?q?3.41?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Docs need the same flag as source code to be able to use fixes from https://gitlab.gnome.org/GNOME/folks/-/merge_requests/49.
---
 docs/meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/docs/meson.build b/docs/meson.build
index 7ace84bd..05264f94 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -35,6 +35,10 @@ valadoc_targets = [
 ]
 
 if eds_backend_enabled
+  if eds_dep.version().version_compare('>=3.41')
+    docs_common_valadoc_flags += ['-D', 'HAS_EDS_3_41']
+  endif
+
   valadoc_targets += {
     'name': 'folks-eds',
     'source_files': eds_backendlib_sources,
-- 
GitLab

